frontend/src/container/QueryBuilder/filters/QueryBuilderSearchV2/QueryBuilderSearchV2.tsx

```diff
@@ -838,6 +838,14 @@ function QueryBuilderSearchV2(
 		[tags],
 	);
 
+	useEffect(() => {
+		// this is required because on change of query tags the Select component re-renders which looses focus.
+		// this was added because Select component was acting inconsistently for similar tags
+		if (selectRef.current && queryTags.length > 0 && isOpen) {
+			selectRef.current?.focus();
+		}
+	}, [queryTags, isOpen]);
```

Comment:
using a prop on the component for focus wouldn't help here.

Reply:
yes used `autoFocus` prop thanks!